highlight.io/components/QuickstartContent/QuickstartContent.tsx

```diff
@@ -663,3 +663,136 @@ export const quickStartContent = {
 		},
 	},
 } as const
+
+export const quickStartContentReorganized = {
+	client: {
+		title: 'Client',
+		sdks: {
```

Comment:
Created this new object to be able to more easily count the SDKs - plan was to refactor the docs page to use this new object over the old one